src/amo/components/Collection/index.js

```diff
@@ -228,7 +245,9 @@ export class CollectionBase extends React.Component<Props> {
           The manager overlay will render full-screen.
           It will be opened/closed based on Redux state.
         */}
-        <CollectionManager collection={collection} />
+        {isLoggedIn ?
```

Comment:
This code wasn't here before, but it seems like we only need to include the `CollectionManager` component if the user is logged in, and this allows that component to be dependent on a logged-in user, which also seems okay.